docs/core/whats-new/dotnet-9/overview.md

```diff
@@ -53,9 +53,57 @@ The .NET 9 SDK introduces _workload sets_, where all of your workloads stay at a
 
 For more information, see [What's new in the SDK for .NET 9](sdk.md).
 
+## AI Building Blocks and Fundamentals
+
+### Microsoft.Extensions.AI & Microsoft.Extensions.VectorData
+
+.NET 9 introduces a unified layer of C# abstractions through [Microsoft.Extensions.AI](https://www.nuget.org/packages/Microsoft.Extensions.AI.Abstractions/) and [Microsoft.Extensions.VectorData](https://www.nuget.org/packages/Microsoft.Extensions.VectorData.Abstractions/). These abstractions facilitate interaction with AI services, including small and large language models (SLMs and LLMs), embeddings, vector stores, and middleware.
+
+### Tokenizers
```

Comment:
I have this under [What's new in ML.NET](https://learn.microsoft.com/en-us/dotnet/machine-learning/whats-new/overview). Should it not be under ML.NET?

Reply:
oh nice... hmmmm yeah good point.... it all kinda blurs together to be honest.